manticore/ethereum.py

```diff
@@ -631,6 +631,26 @@ def did_evm_write_storage_callback(self, state, address, offset, value):
         state.context.setdefault('{:s}.initialized_storage'.format(self.name), set()).add((address, offset))
 
 
+class DetectMultipleSends(Detector):
+    '''
+    Detects multiple sends to the same address within a transaction
+    '''
+    def will_evm_execute_instruction_callback(self, state, instruction, arguments):
```

Comment:
Should we do this after the fact and only if the transaction was successful?
Also not sure if it is important for the current goal but you can send value with CREATE, and SELFDESTRUCT :) and DELEGATECALL 
all open_transactions + delegatecall